openfisca_core/variables.py

```diff
@@ -56,6 +56,17 @@ def to_column(self, tax_benefit_system):
             except ValueError:
                 raise ValueError(u"Incorrect 'end' attribute format in '{}'. 'YYYY-MM-DD' expected where YYYY, MM and DD are year, month and day. Found: {}".format(self.name, end).encode('utf-8'))
 
+        reference = self.attributes.pop('reference', None)
+        if reference:
+            if isinstance(reference, unicode) or isinstance(reference, str):
```

Comment:
`isinstance(reference, basestring)` is equivalent to `isinstance(reference, unicode) or isinstance(reference, str)`